services/common/src/tests/mocks/dataMocks.tsx

```diff
@@ -8062,7 +8062,7 @@ export const PROJECT_SUMMARY_AUTHORIZATION_TYPES_TRANSFORMED = [
       { code: "AIR_EMISSIONS_DISCHARGE_PERMIT", description: "Air emissions discharge permit" },
       { code: "EFFLUENT_DISCHARGE_PERMIT", description: "Effluent discharge permit" },
       { code: "REFUSE_DISCHARGE_PERMIT", description: "Refuse discharge permit" },
-      { code: "MUNICIPAL_WASTEWATER_REGULATION", description: "Municipal Wastewater Regulation" },
```

Comment:
Personally I prefer removing these kinds of things and re-adding them in the future.  Comments have a habit of staying in the code long past where we remember why they're there.  Not a huge deal, but who knows when we'll add this functionality back in.

Reply:
Sounds good, the code that was commented out is now removed.